src/lib/jsx.js

```diff
@@ -117,7 +117,13 @@ export function jsxRender(template : string, renderers : { [string] : (string) =
                 return text;
             }
 
-            return renderers.text(text);
+            // handle break here
+            let regex = /\<br\>/;
```

Comment:
Do we need this logic? Wouldn't line `113` already call the break function?

Reply:
Yes, with the current standpoint.

Reply:
👍